bundles/org.openhab.binding.meteoalerte/src/main/resources/OH-INF/i18n/meteoalerte.properties

```diff
@@ -0,0 +1,188 @@
+thing-type.config.meteoalerte.department.department.option.MEUSE = Meuse
+thing-type.config.meteoalerte.department.department.option.MORBIHAN = Morbihan
+thing-type.config.meteoalerte.department.department.option.MOSELLE = Moselle
+thing-type.config.meteoalerte.department.department.option.NIEVRE = Nièvre
+thing-type.config.meteoalerte.department.department.option.NORD = Nord
+thing-type.config.meteoalerte.department.department.option.OISE = Oise
+thing-type.config.meteoalerte.department.department.option.ORNE = Orne
+thing-type.config.meteoalerte.department.department.option.PARIS = Paris
+thing-type.config.meteoalerte.department.department.option.PAS-DE-CALAIS = Pas de Calais
+thing-type.config.meteoalerte.department.department.option.PUY-DE-DOME = Puy de Dôme
+thing-type.config.meteoalerte.department.department.option.PYRENEES-ATLANTIQUES = Pyrénées Atlantiques
+thing-type.config.meteoalerte.department.department.option.PYRENEES-ORIENTALES = Pyrénées Orientales
+thing-type.config.meteoalerte.department.department.option.RHONE = Rhône
+thing-type.config.meteoalerte.department.department.option.SAONE-ET-LOIRE = Saône et Loire
+thing-type.config.meteoalerte.department.department.option.SARTHE = Sarthe
+thing-type.config.meteoalerte.department.department.option.SAVOIE = Savoie
+thing-type.config.meteoalerte.department.department.option.SEINE-ET-MARNE = Seine et Marne
+thing-type.config.meteoalerte.department.department.option.SEINE-MARITIME = Seine Maritime
+thing-type.config.meteoalerte.department.department.option.SEINE-SAINT-DENIS = Seine Saint Denis
+thing-type.config.meteoalerte.department.department.option.SOMME = Somme
+thing-type.config.meteoalerte.department.department.option.TARN = Tarn
+thing-type.config.meteoalerte.department.department.option.TARN-ET-GARONNE = Tarn et Garonne
+thing-type.config.meteoalerte.department.department.option.TERRITOIRE\ DE\ BELFORT = Territoire de Belfort
+thing-type.config.meteoalerte.department.department.option.VAL-D'OISE = Val D'Oise
+thing-type.config.meteoalerte.department.department.option.VAL-DE-MARNE = Val de Marne
+thing-type.config.meteoalerte.department.department.option.VAR = Var
+thing-type.config.meteoalerte.department.department.option.VAUCLUSE = Vaucluse
+thing-type.config.meteoalerte.department.department.option.VENDEE = Vendée
+thing-type.config.meteoalerte.department.department.option.VIENNE = Vienne
+thing-type.config.meteoalerte.department.department.option.VOSGES = Vosges
+thing-type.config.meteoalerte.department.department.option.YONNE = Yonne
+thing-type.config.meteoalerte.department.department.option.YVELINES = Yvelines
+thing-type.config.meteoalerte.department.refresh.label = Fréquence de rafraichissement
+thing-type.config.meteoalerte.department.refresh.description = Période d'actualisation des données en minutes.
+
+# channel types
+
+channel-type.meteoalerte.avalanches.label = Etat Avalanches
```

Comment:
Please regenerate the file, you should not have "Etat".

Reply:
Yes, forgot it. Done